### PR TITLE
opensips: Move all oss-fuzz testing into opensips/opensips

### PR DESCRIPTION
**Pull request by liviuchircu**
Specifically, into the
https://github.com/OpenSIPS/opensips/tree/master/test/fuzz directory. Build is now invoked through the `oss-fuzz-build.sh` script.

This move [was suggested](https://github.com/google/oss-fuzz/pull/7469#pullrequestreview-924998018) by @jonathanmetzman a few months ago, in #7469.

### CODE REVIEW

File: projects/opensips/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2022 Google LLC
```
Comment (DavidKorczynski) [inline projects/opensips/Dockerfile:1]: nit: please leave copyright year 2021

File: projects/opensips/build.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eu
-# Copyright 2021 Google LLC
+# Copyright 2022 Google LLC
```
Comment (DavidKorczynski) [inline projects/opensips/build.sh:2]: ditto

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]
awesome!

File: projects/opensips/Dockerfile (hunk 1 above)
Comment (liviuchircu) [reply projects/opensips/Dockerfile]: Sure!  Our of curiosity, is it because A) 2021 is correct, or B) a single change per PR is better?

Comment (DavidKorczynski) [reply projects/opensips/Dockerfile]: A) 2021 is correct

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**